scripts/source-maintenance.sh

```diff
@@ -35,6 +38,10 @@ while [ $# -ge 1 ]; do
         KeepGoingDirective=$1
         shift
         ;;
+    --enable-astyle)
```

Comment:
This option does not enable astyle or formatting. Formatting with astyle is enabled with or without this option. AFAICT, the option should be renamed to `--with-astyle-version`.

However, it would actually be better to replace this with `--with-astyle` (i.e., to specify the name of the executable) so that one does not have to install astyle globally or add it to `$PATH` to use it.

Reply:
done